pkg/monitors/prometheus/velero/metadata.yaml

```diff
@@ -0,0 +1,47 @@
+monitors:
+- monitorType: prometheus/velero
+  doc: |
+    This monitor gets metrics from 
+    [Velero](https://github.com/vmware-tanzu/velero).
+    It is a wrapper around the [prometheus-exporter](./prometheus-exporter.md) 
+    monitor that provides a restricted but expandable set of metrics.
+
+    <!--- SETUP --->
+    ### Velero configuration
+
+    The Helm chart automatically enable prometheus metrics
+    for Velero (see [this PR](https://github.com/helm/charts/pull/19595/files))
+    So there is nothing to do.
+
+    ### Agent configuration
+
+    This is recommended to use service discovery:
+
+    ```yaml
+    monitors:
+    - type: prometheus/velero
+      discoveryRule: container_image =~ "velero" && port == 8085
+      port: 8085
+    ```
+
+  metrics:
```

Comment:
Is this just a subset of the metrics exposed by Valero or is this all of them?

Reply:
yes, just a subset, velero expose lot of metrics (in addition to default golang metrics).